src/main/java/htsjdk/samtools/DuplicateScoringStrategy.java

```diff
@@ -87,9 +91,23 @@ public static short computeDuplicateScore(final SAMRecord record, final ScoringS
                     }
                     break;
                 case RANDOM:
-                    score += (short) (hasher.hashUnencodedChars(record.getReadName()) >> 16);
+                    // start with a random number between Short.MIN_VALUE/4 and Short.MAX_VALUE/4
+                    score += (short) (hasher.hashUnencodedChars(record.getReadName()) & 0b11_1111_1111_1111);
```

Comment:
Oh, I remember why *you* did this :smile:, you wanted the same score for both mate of the same query name....so nextInt will not do. I'll add a comment to this effect.